core/services/internal/mocks/application.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.0.3. DO NOT EDIT.
```

Comment:
Any apparent reason why this isn't grouped with the other mocks in `core/internal/mocks`?

Reply:
I didn't change anything related to this mock